build.gradle.kts

```diff
@@ -13,6 +13,8 @@ repositories {
 }
 
 dependencies {
+  implementation("org.testng:testng:7.1.0")
```

Comment:
jeżeli już to `testImplementation("org.testng:testng:7.1.0")` ale strzelam ze IDE Ci to wstawilo, wiec weryfikuj to co sie dzieje automatycznie, bo tak sie balagan w zaleznosciach projektu zrobil